utils/gyb.py

```diff
@@ -304,7 +309,7 @@ def splitGybLines(sourceLines):
     dedents = 0
     try:
         for tokenKind, tokenText, tokenStart, (tokenEndLine, tokenEndCol), lineText \
-            in tokenize.generate_tokens(sourceLines.__iter__().next):
+            in tokenize.generate_tokens(partial(next, sourceLines.__iter__())):
```

Comment:
I've gone ahead and switched over to the `lambda` syntax (see: dc70d6d06119b72f3733da56fccb3c2d66082d8a). As a newbie to Python this feels like a lateral move but I defer to those who have been there before me.

@frewsxcv thanks for showing me some Python! :beers: